docs/communicate-with-javascript.md

```diff
@@ -30,6 +32,8 @@ javascript add : int -> int -> int = {|function(x,y){
 But this would break compatibility with OCaml, which is one of the main goals of
 Melange.
 
+I would explain this positively instead. Instead of what melange needs to add, can we explain what’s the actual mechanism? Like OCaml with C bindings: https://v2.ocaml.org/manual/intfc.html#:~:text=User primitives are,C-function-name -->
```

Comment:
I agree this is a bit confusing. I would probably remove all of it and just write a small intro saying something like

> Attributes and extension nodes allow to annotate OCaml code to help with code generation, or express functionality that is not supported by default in the language. They look like this:
> `let add = [%mel.raw "a + b"]`
> `let [@mel.as "POST"] js_post = post`
> We will see both kinds in the next subsections.

wdyt?